sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -2225,11 +2231,13 @@
   /**
    * @since 4.5
    */
+  // TODO should be renamed to TECHNICALDEBT_RATIO_KEY = "technicaldebt_ratio"
   public static final String SQALE_DEBT_RATIO_KEY = "sqale_debt_ratio";
 
   /**
    * @since 4.5
    */
+  // TODO should be renamed to TECHNICALDEBT_RATIO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)